### PR TITLE
#278 - Extracted FilePackageHeader from FilePackage

### LINKED ISSUE #278: PrimaryOutputTest.java:88-90: Introduce class from this...
The puzzle `241-a350bc6d` from #241 has to be resolved: 

https://github.com/artipie/rpm-adapter/blob/790b309d40175b7014e4e58ca197f2e7f0120b52/src/test/java/com/artipie/rpm/pkg/PrimaryOutputTest.java#L88-L90

The puzzle was created by olenagerasimova on 18-Jun-20. 

Estimate: 30 minutes,  role: DEV. 

If you have any technical questions, don't ask me, submit new tickets instead. The task will be \"done\" when the problem is fixed and the text of the puzzle is _removed_ from the source code. Here is more about [PDD](http://www.yegor256.com/2009/03/04/pdd.html) and [about me](http://www.yegor256.com/2017/04/05/pdd-in-action.html).

### PR DESCRIPTION
**Pull request by olegmoz**
Closes #278 
Extracted `FilePackageHeader` from `FilePackage`, added usage in tests to reduce code duplication

### CODE REVIEW

File: src/main/java/com/artipie/rpm/pkg/FilePackageHeader.java [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+import java.util.logging.Level;
+import org.redline_rpm.ReadableChannelWrapper;
+import org.redline_rpm.Scanner;
+import org.redline_rpm.header.Format;
+import org.redline_rpm.header.Header;
+
+/**
+ * Header of RPM package file.
+ *
+ * @since 0.11
```
Comment (olenagerasimova) [inline src/main/java/com/artipie/rpm/pkg/FilePackageHeader.java:41]: @olegmoz since 0.10

**Comment by 0crat** [comment]
> @0crat assign me ([here](https://github.com/artipie/rpm-adapter/issues/293#issuecomment-649351584))

@olenagerasimova This pull request [#293](https://github.com/artipie/rpm-adapter/issues/293) is assigned to @olenagerasimova[/z](https://www.0crat.com/u/olenagerasimova), here is [why](https://www.0crat.com/footprint/CT2E6TK9B/8fbee93c-9a20-45af-a422-f6c35b85fc74); the budget is 15 minutes, see [§4](http://www.zerocracy.com/policy.html#4); please, read [§27](http://www.zerocracy.com/policy.html#27) and when you decide to accept the changes, inform @g4s8[/z](https://www.0crat.com/u/g4s8) (the architect) right in this ticket; if you decide that this PR should not be accepted ever, also inform the architect; this [blog post](http://www.yegor256.com/2015/02/09/serious-code-reviewer.html) will help you understand what is expected from a code reviewer; there will be no monetary reward for this job

File: src/main/java/com/artipie/rpm/pkg/FilePackageHeader.java (hunk 1 above)
Comment (olegmoz) [reply src/main/java/com/artipie/rpm/pkg/FilePackageHeader.java]: @olenagerasimova fixed, thanks

**Comment by 0crat** [comment]
Job was finished in 2 hours, bonus for fast delivery is possible (see [§36](http://www.zerocracy.com/policy.html#36))

**Pull request merged**